### ISSUE TITLE
Human Campaign Mission 5

### ISSUE THREAD

**Issue by timfel** [Wargus/wargus#84]
Cannot start mission 5 (I see the video, the speech but it blocks when files are loading).

I have version 2.2.6 x86 + the trunk fix (from a link somewhere in these forums, I needed the fix only for peasants tree cutting fix!).

No error files are created in stratagus folder (which is strange) so I do not know/cannot know what's wrong...  (Windows 7)

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2012-08-18T11:44:04Z
 * owner: vortaka
 * assignee: joris-dauphin
 * the launchpad url was https://bugs.launchpad.net/bugs/1038421

**Comment by timfel** [comment]
(by vortaka)
If I'm reading right, the last thing he looks for is an ai script that he doesn't find!
(Also have a post on stratagus forum)

**Comment by timfel** [comment]
(by sshsteph007)
I noticed the same bug in level 5, fixed it myself by editing the file
 "campaigns/human/level05h_c.sms" which is located  in the installation data directory
(/usr/local/share/games/stratagus/wargus/ or /usr/share/games/stratagus/wargus/).
In this file look for the line
  "DefineAi("hum-05", "*", "hum-05", AiHuman05)"
and add the following line after it
  "DefineAi("orc-03", "*", "orc-03", AiPassive)"
So that there are two AIs defined. The "orc-03" seemed a passive ai too me and
with it the level is loaded correctly and playable.

I suspect the problem is in wartool converting the original data or the original data already
had this bug. Just for convenience, I used wartool version v1.3
